keras_cv/backend/config.py

```diff
@@ -21,7 +21,8 @@ def detect_if_tensorflow_uses_keras_3():
         # Note that only recent versions of keras have a `version()` function.
         if hasattr(keras, "version") and keras.version().startswith("3."):
             return True
-    except:
+    except Exception as e:
+        raise e
```

Comment:
Unrelated change, will revert in the next batch of commits

Reply:
okay